pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -144,7 +144,22 @@ self: super: {
     then dontCheck (overrideCabal super.hakyll (drv: {
       testToolDepends = [];
     }))
-    else super.hakyll;
+    else overrideCabal super.hakyll {
+      patches = [
```

Comment:
I think this only applies the patches if built on non-darwin?  Is this logic correct?